src/typings/react-native-windows.d.ts

```diff
@@ -0,0 +1,36 @@
+/**
+* react-native-windows.d.ts
+*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT license.
+*
+* Type definition file for React Native Windows only components and modules
+* Definitions for extensions pertaining to existing React Native components are merged into the reaxt-native.d.ts file.
+*/
+
+declare module 'react-native-windows' {
+    //
+    // React
+    // ----------------------------------------------------------------------
+
+    import React = require('react');
+    import RN = require('react-native');
+
+    //
+    // Focusable view related declarations
+    // ----------------------------------------------------------------------
+    interface FocusableProps extends RN.ViewProps {
+        isTabStop?                      : boolean;
+        tabIndex?                       : number;
+        tabNavigation?                  : string; // enum( 'local', 'cycle', 'once' );
```

Comment:
>string; // enum( 'local', 'cycle', 'once' ); [](start = 42, length = 44)

I think you can use string types here:

`tabNavigation?: 'local' | 'cycle' | 'once'

Reply:
Will do. This trick doesn't seem to be used elsewhere, unfortunately.

---
In reply to: [159357725](https://github.com/Microsoft/reactxp/pull/404#discussion_r159357725) [](ancestors = 159357725)